doc/sedcli.8

```diff
@@ -94,6 +94,18 @@ MBR shadow should be enabled for the PBA to take effect.
 .IP "\fB\-B -d <device> -r {1|0}\fR or \fB\-\-block_sid --device <device> --hwreset {1|0}\fR"
 Issue BlockSID authentication command with Clear Event flag via --hwreset option.
 
+.IP "\fB\-U -d <device> -u <userN> -t <accesstype> [-l <locking-range>]\fR or \fB\-\-add-user-to-lr --device <device> --user <userN> --accesstype {RO|RW|LK} [--locking-range <locking-range>]\fR"
+.IP
+Add users to the Locking Ranges with the following access modes:
+.IP
+1. Read-Only(RO) - user can only read the data from the disk.
+.IP
+2. Read-Write(RW) - user can read/write data from/to the disk.
+.IP
+3. Locked(LK) - data is locked, user can NOT read/write data from/to the disk.
+.IP
+With a max. of 9 Locking Ranges available on a device, the value of N ranges from 1 to 9.
```

Comment:
spec at minimum require 8 + 1 ranges. what if device supports more than 8+1 locking ranges?

Reply:
two options here.
1. do the level 0 discovery and use the SED Opal v2.00 features' Admin Authorities LSP Supported, User Authorities LSP Supported values to check the user provided values.
2. Ask the user to check those values via discover 0 command